code/index.html

```diff
@@ -8,30 +8,46 @@
     <title>Technigo Project Cluedo</title>
   </head>
   <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
-      </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
-      </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
-      </article>
-    </section>
-
-    <h1 id="mystery"></h1>
-
-    <button>REVEAL THE CRIME</button>
-
+    <main>
+      <!--Hero image and information-->
+      <img class="hero-image" src="picture6.jpg" alt="hero-image-crystalchandeliers">
```

Comment:
Good practice to use ./ in the src  for images i.e. ./picture6.jpg

Reply:
Yes, good point!

Reply:
Fixed :)